OSPSuite.Presentation.Importer/Core/DataFormat/ColumnsDataFormat.cs

```diff
@@ -0,0 +1,153 @@
+﻿
+using System.Text.RegularExpressions;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using NPOI.SS.Util;
+using System;
+
+namespace OSPSuite.Presentation.Importer.Core.DataFormat
+{
+   public class ColumnsDataFormat : IDataFormat
+   {
+      public string Name => "ColumnsFormat";
+
+      public string Description => "Simple format with data identified by column data...";
+
+      public IList<DataFormatParameter> Parameters { get; private set; }
+
+      public bool CheckFile(IUnformattedData data)
+      {
+         if (data.Headers.Where(h => h.Value.Level == ColumnDescription.MeasurementLevel.NUMERIC).Count() < 2)
```

Comment:
Explain what 2 means.. It's ok to use magic numbers but comments are required so that we understand what that means in 6 months

Reply:
At least a mapping for the time and the observation... Keep in mind that this is some kind of very simple format just to have the whole picture but until we agree on the formats this is not going to be thoroughly implemented